astronomer/providers/google/cloud/example_dags/example_dataproc.py

```diff
@@ -243,20 +190,21 @@
     # [END howto_DataprocSubmitJobOperatorAsync]
     # [START how_to_cloud_dataproc_delete_cluster_operator]
     delete_cluster = DataprocDeleteClusterOperator(
-        task_id="delete_cluster", project_id=PROJECT_ID, cluster_name=CLUSTER_NAME, region=REGION
+        task_id="delete_cluster",
+        project_id=PROJECT_ID,
+        cluster_name=CLUSTER_NAME,
+        region=REGION,
+        trigger_rule="all_done",
     )
     # [END how_to_cloud_dataproc_delete_cluster_operator]
     # [START howto_delete_buckettask]
     delete_bucket = GCSDeleteBucketOperator(
         task_id="delete_bucket",
         bucket_name=BUCKET,
+        trigger_rule="all_done",
     )
     # [END howto_delete_buckettask]
 
-    create_cluster >> scale_cluster >> create_bucket
-    scale_cluster >> create_workflow_template >> trigger_workflow >> delete_cluster
-    scale_cluster >> hive_task >> delete_cluster >> delete_bucket
-    scale_cluster >> pig_task >> delete_cluster >> delete_bucket
-    scale_cluster >> spark_sql_task >> delete_cluster >> delete_bucket
-    scale_cluster >> spark_task >> delete_cluster >> delete_bucket
-    scale_cluster >> hadoop_task >> delete_cluster >> delete_bucket
+    create_cluster >> create_bucket
+    create_cluster >> pig_task >> hive_task >> delete_cluster >> delete_bucket
```

Comment:
Running it in less parallel so there will be minimal chance to fail because of unavailable resource